src/component/Nav.js

```diff
@@ -1,4 +1,4 @@
-import 'bootstrap/dist/css/bootstrap.min.css';  
+//import 'bootstrap/dist/css/bootstrap.min.css';  
 import {Nav, Navbar, Container} from 'react-bootstrap';  
```

Comment:
`import { Nav, Navbar, Container } from 'react-bootstrap'; `
space before and after curly braces

Reply:
still not changed?